### ISSUE TITLE
npm install broken

### ISSUE THREAD

**Issue by konstantinkoeppe** [zebzhao/Angular-QueryBuilder#10]
It seems like npm install is currently broken.

When creating a new project using all defaults of `npm init` and adding this to the project via `npm install angular2-query-builder --save` all you get is this:

![image](https://user-images.githubusercontent.com/23746325/35738597-9c4cef2e-082f-11e8-950e-754adb69bf29.png)

Am I missing something here?

**Comment by zebzhao** [comment]
You're right. Seems like I skipped building the library before publishing the last release. This should be fixed in the latest release. Thanks for reporting it.
